xenon/cogs/builder.py

```diff
@@ -125,7 +125,7 @@ def _create_embed(self):
         page_options = self.pages[self.page - 1]
         embed = self.ctx.em("", title="Server Builder")["embed"]
         embed.title = page_options["name"].title()
-        embed.set_footer(text="Enable / Disable options with the reactions and click ✅ when you are done")
+        embed.set_footer(text="Enable / Disable options with the reactions and click ✅ when you are done.")
```

Comment:
That's a footer. I don't think a . at the end of the sentence is a good idea here.